### PR TITLE
reorganized content in ts and FAQs section and added new CICS questions

### PR DESCRIPTION
**Pull request by RadhaV2020**
Signed-off-by: RadhaV2020 <63616898+RadhaV2020@users.noreply.github.com>

### CODE REVIEW

**Comment by RadhaV2020** [comment]
Some minor issues with hyperlinks exist in this PR. Will be fixed in the next commit

**Comment by RadhaV2020** [comment]
fixed the hyperlink issues, moved the CICS troubleshooting question under the TS section per the CICS team's ask.

File: source/Troubleshooting/errorsandmessages.rst [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+``zos_copy`` to **true**.
+
+Troubleshooting Collections
+===========================
+
+If I run into a problem when using an Ansible collection for IBM Z, how should I seek support?
+----------------------------------------------------------------------------------------------
+* For issues related to the Ansible collections, raise a GitHub issue against the appropriate collection repository:
+
+  * `IBM z/OS core <https://github.com/ansible-collections/ibm_zos_core/issues/new/choose>`_
```
Comment (itsBryantP) [inline source/Troubleshooting/errorsandmessages.rst:162]: update to include references to all collections that are part of RHACCIZ

File: source/faqs/faqs.rst [hunk 2]
```diff
@@ -218,156 +270,16 @@ Refer to the official testing  `strategies recommended by Ansible`_.
...
-
-
-Why does my managed z/OS system pre-login prompt cause the ``zos_copy`` and ``zos_fetch`` modules to fail?
-----------------------------------------------------------------------------------------------------------
-
-By default, ``SFTP`` redirects **pre-login** prompts to system **stderr**,
-which ``zos_copy`` interprets as a failure. You can bypass this behavior and
-ignore stderr content by setting **ignore_sftp_stderr** parameter of
-``zos_copy`` to **true**.
+* `IBM z/OS core <https://github.com/ansible-collections/ibm_zos_core/issues/new/choose>`_
```
Comment (itsBryantP) [inline source/faqs/faqs.rst:281]: update to include references to all collections that are part of RHACCIZ

**Review by itsBryantP — CHANGES_REQUESTED** [review]

**Review by itsBryantP — APPROVED** [review]

**Pull request merged**